contrib/endpoints/src/api_manager/proto/server_config.proto

```diff
@@ -139,6 +142,12 @@ message ApiAuthenticationConfig {
   bool force_disable = 1;
 }
 
+message Mixer_options {
+  // If not present, default to use service_control. When Envoy/esp talks to
+  // Mixer, has to specify this field.
+  string mixer_server_name = 1;
```

Comment:
Just call it "mixer_server".
add comment
// For envoy, it is the cluster name for mixer server.